### DISCUSSION TITLE
Welcome to DADN_TTNT Discussions!

### DISCUSSION THREAD

**Discussion by khoavukhoa3004** [khoavukhoa3004/DADN_TTNT#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nguyencharlie** [comment]
Ask: Việc thêm dữ liệu (đèn, quạt, cảm biến) sẽ được thực hiện như thế nào? Mọi người cứ đưa ra ý kiến nha.

Me: Thêm vào MongoDB để lưu trữ, nếu được thì thêm vào luôn vào Adafruit (muốn chạy được thì phải gắn thiết bị/cảm biến vào, và cái này thì sẽ do bên Gateway và Yolo:bit xử lý).

**Reply by khoavukhoa3004** [reply]
Tui nghĩ mik sẽ thêm dữ liệu vào adafruit trc, sau đó nếu nhận đc response thành công thì ms đẩy dữ liệu lên mongodb

**Comment by nguyencharlie** [comment]
Ask (Back-end): team mình sẽ sử dụng API thông qua MQTT.js hay sử dụng chính API của Adafruit?

Me: hôm trước (trước bữa thứ 5) tui với Đăng có làm về MQTT.js. Nhưng vào bữa thứ 5 thì Minh có làm về Adafruit API.

**Comment by nguyencharlie** [comment]
Ask (Device & Sensor): Đối với các thiết bị như quạt, đèn có nên thêm một Button là CHẾ ĐỘ (Mode) không? Và nếu có thì chức năng cho từng thiết bị đó sẽ như thế nào?

**Comment by nguyencharlie** [comment]
Ask (Device & Sensor): Như mọi người đã biết, thì trong UI của đèn và quạt, còn thể hiện những thông số như Nhiệt độ trong nhà, Nhiệt độ ngoài trời, và Giá trị của cảm biến ánh sáng. Nhưng ở bữa thứ 5 (13/4) thì ở trang Home cũng có những thông số đó. Vậy thì ở UI của đèn và quạt có còn cần thiết nữa không? và nếu cần thì nên để như thế nào cho phù hợp?
